### PR TITLE
update frontend for jdbc compatibility

### PR DESCRIPTION
**Pull request by daviszhen**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #2386 

**What this PR does / why we need it:**

Updates:
1, update setting variable  in session context.
2, fix commend field listing for tae.

The jdbc and mysql administrator gui tools like DBeaver, mysql workbench read and update some system variables after the connection with mysql was established or during the iteration with mysql. The jdbc compatibilty has relationship with these system variables that should be correct, updatable. The mo designs the mechanism to maintain these variables like changing,reading and persisting the values of system variables in suitable context. 

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Review by fengttt — COMMENTED** [review]
Can you please update your checkin message?    What was the issue (what compatibility issue?) and what you did?    A few sentences should be good.

**Comment by daviszhen** [comment]
> Can you please update your checkin message? What was the issue (what compatibility issue?) and what you did? A few sentences should be good.

I have updated the message

File: pkg/frontend/mysql_cmd_executor.go [hunk 1]
```diff
@@ -575,20 +577,31 @@ func (mce *MysqlCmdExecutor) handleChangeDB(db string) error {
...
+		val = ses.GetDatabaseName()
+		if val == "" {
+			val = "NULL"
+		}
+	case "current_user":
+		val = ses.GetUserName()
+		host, _ := proto.Peer()
+		val = fmt.Sprintf("%s@%s", val, host)
+	case "connection_id":
+		val = proto.ConnectionID()
```
Comment (jianwan0214) [inline pkg/frontend/mysql_cmd_executor.go:602]: Don't you need to add default branch here?

**Comment by daviszhen** [comment]
it is outdated and replaced by #2829

**Pull request closed without merging**